lib/rubrowser/a.rb

```diff
@@ -0,0 +1,23 @@
+class A
+  def initialize
+    B.do
+  end
+
+  def self.do; end
+end
+
+class B
```

Comment:
I'd love to find another way to test the app's circular functionality without including a file that does nothing